### DISCUSSION TITLE
mpv no longer has libcaca support enabled

### DISCUSSION THREAD

**Discussion by NightMachinery** [Homebrew/discussions#3072] [Tap maintenance and Homebrew development]
`mpv` had optional libcaca support, but it seems after Homebrew removed compile options, this support was simply disabled.

I think it's more sensible to now enable this support for everyone. It's not like having an additional dependency is so bad.

`caca` will appear in this list if it is compiled in:
```
❯ /usr/local/bin/mpv --vo=help                                                1
Available video outputs:
  libmpv           render API for libmpv
  gpu              Shader-based GPU Renderer
  null             Null video output
  image            Write video frames to image files
  tct              true-color terminals
```

**Comment by SMillerDev** [comment]
> It's not like having an additional dependency is so bad.

It increases the maintenance work required since it's more parts that can break so it's something we're careful about.

You're also the first to suggest this AFAIR 4 years after the fact so it doesn't seem that popular.

**Reply by carlocab** [reply]
> You're also the first to suggest this AFAIR 4 years after the fact so it doesn't seem that popular.

The `libcaca` option won't have been too popular when it was available too, since the option removal was based on analytics.

**Comment by NightMachinery** [comment]
[**@
          carlocab
        **](https://github.com/carlocab) commented on [Mar 13, 2022, 10:43 AM GMT+3:30](https://github.com/Homebrew/discussions/discussions/3072#discussioncomment-2348994 "2022-03-13T07:13:21Z - Replied by Github Reply Comments"):
> > It's not like having an additional dependency is so bad.
> 
> It increases the maintenance work required since it's more parts that can break so it's something we're careful about.
> 
> You're also the first to suggest this AFAIR 4 years after the fact so it doesn't seem that popular.

Okay. `wget` also doesn't support PCRE regex, which is much more essential. Should I open an issue for that?
